### PR TITLE
[SOAR-10660] Rapid7 IntSights rename to Threat Command

### PR DESCRIPTION
**Pull request by ablakley-r7**
### Description

Describe the proposed changes:

  - Rename Plugin to Threat Command

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Review by ablakley-r7 — COMMENTED** [review]
Worth noting remaining IntSights references. plugin.spec.yaml lines 255. Update to output will require major version increase. Descriptions for Account ID and API Key should be reviewed alongside Threat Command documentation for consistency.

File: plugins/rapid7_intsights/plugin.spec.yaml [hunk 1]
```diff
@@ -2,20 +2,19 @@ plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
 name: rapid7_intsights
-title: Rapid7 IntSights
-description: Rapid7 IntSights is disrupting external threat intelligence with a combination of human and automated collection, intelligent analysis, and strategic threat hunting that turns the clear, deep, and dark webs into an intelligence resource that any company can deploy
-version: 3.2.0
+title: Threat Command
+description: Threat Command by Rapid7 is disrupting external threat intelligence with a combination of human and automated collection, intelligent analysis, and strategic threat hunting that turns the clear, deep, and dark webs into an intelligence resource that any company can deploy
```
Comment (cmcnally-r7) [inline plugins/rapid7_intsights/plugin.spec.yaml:6]: Quick question, is it supposed to be Threat Command by Rapid7 or just Threat Command?

File: plugins/rapid7_intsights/plugin.spec.yaml [hunk 2]
```diff
@@ -2,20 +2,19 @@ plugin_spec_version: v2
...
+version: 3.2.1
 supported_versions: ["2.4.0"]
 vendor: rapid7
 support: rapid7
 status: []
 resources:
-  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/rapid7_intsights
+  source_url: https://github.com/rapid7/insightconnect-polugins/tree/master/plugins/rapid7_intsights
```
Comment (cmcnally-r7) [inline plugins/rapid7_intsights/plugin.spec.yaml:13]: I'd undo this change, it must be a typo

**Review by cmcnally-r7 — COMMENTED** [review]
Just one small typo to fix up there

File: plugins/rapid7_intsights/plugin.spec.yaml (hunk 1 above)
Comment (ablakley-r7) [reply plugins/rapid7_intsights/plugin.spec.yaml:6]: Threat Command by Rapid7 is the approved wording for this description.

**Review by igorski-r7 — APPROVED** [review]

**Review by cmcnally-r7 — APPROVED** [review]

**Review by blaxminarayan-r7 — APPROVED** [review]
